### ISSUE TITLE
Reverse display of trees for RTL text

### ISSUE THREAD

**Issue by mr-martian** [gucorpling/midas-loop-ui#4]
The first sentence of Exodus (with random head service):
![image](https://user-images.githubusercontent.com/5350777/180104874-138efb08-f4c1-466f-b085-70acd165b011.png)

Text: ` וְאֵ֗לֶּה שְׁמֹות֙ בְּנֵ֣י יִשְׂרָאֵ֔ל הַבָּאִ֖ים מִצְרָ֑יְמָה אֵ֣ת יַעֲקֹ֔ב אִ֥ישׁ וּבֵיתֹ֖ו בָּֽאוּ׃`

It would be nice if the tree would display RTL to match the text rather than LTR as it is currently.

Also, is there a reason for never displaying supertokens?

**Comment by amir-zeldes** [comment]
I'm not sure how complex RTL support might be, I'll leave that to @lgessler and @lauren-lizzy-levine to answer, but regarding super tokens, something looks weird here - the tokens that are part of MWTs are not rendered at all and instead we see underscores... 

I think the datamodel does actually support MWT, so this may be a bug. But aside from the underscores, I don't think that MWTs should necessarily be visualized in this view, since they can't be annotated and would mainly be distracting to visualize. If we end up implementing support for word tokenization in the segmentation module, it might make more sense to figure out MWT rendering there (since segmentation is the basic annotation operation with which you would edit MWTs - they have no POS, deprel or lemma)

**Comment by mr-martian** [comment]
The underscores in this case are because I initially didn't have surface forms for components of MWTs in Ancient_Hebrew-PTNK, so that's an issue with my data rather than your code.

**Comment by amir-zeldes** [comment]
Oh, phew, that's a relief to hear, then the system is working as expected. We should leave this issue open for the RTL thing, but for MWTs I don't think we'll add them in this particular view in the future (the segmenter may be a good place to represent them at some point).

**Comment by lgessler** [comment]
Hmm, suspect this would require medium effort. The fix would require detecting when a sentence is RTL and modifying the SVG rendering to accommodate. I unfortunately don't think we'll get to this soon, so in the meantime a transcription into an LTR script could be a workaround.

**Comment by amir-zeldes** [comment]
Agreed, this is not a priority while there are a lot of other things to do on the system. For posterity I'll just add that I think RTL is implemented in Arborator now, and I remember dimly implementing it once before it was supported in the official arborator code, so if the algorithm is the same as Arborator's I probably have written down somewhere what code to change (it wasn't tons).
